specs/validators/numericality-spec.js

```diff
@@ -293,4 +293,30 @@ describe("validators.numericality", function() {
       , value = "foo";
     expect(numericality(value, options)).toBe(message);
   });
+
+  describe("noExtraZero", function() {
+    it("disallows extra zeros", function () {
+        expect(numericality("1.0000000", {noExtraZero: true})).toBe(
+            "must be no extra zeros"
+        );
+        expect(numericality("1.00", {noExtraZero: true})).toBe(
```

Comment:
Refer to the above floating point numbers, I am not sure how should I validate such number?

Should I allow all trailing zeros for floating point numbers?